frontend/src/app/modules/boards/board/board-actions/version/version-board-header.component.ts

```diff
@@ -28,6 +28,7 @@
 import {Component, Input} from "@angular/core";
 import {VersionResource} from "core-app/modules/hal/resources/version-resource";
 import {I18nService} from "core-app/modules/common/i18n/i18n.service";
+import {PathHelperService} from "core-app/modules/common/path-helper/path-helper.service";
 
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/e056d70ed7d3ea289a5da1e1b836aec8cd40cf64/frontend/src/app/modules/boards/board/board-actions/version/version-board-header.component.ts#L33"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>